### PR TITLE
Muelu: fix experimental nightlies

### LINKED ISSUE #9725: MueLu: compile errors on experimental dashboard
## Bug Report
@trilinos/muelu 

### Description
There are a number of compile failures under the experimental track.

https://testing.sandia.gov/cdash/index.php?project=Trilinos&subproject=MueLu#!#Experimental

### PR DESCRIPTION
**Pull request by jhux2**
This is to fix #9725, build errors in MueLu's experimental dashboard.

### CODE REVIEW

File: packages/muelu/test/scaling/MatvecKernelDriver.cpp [hunk 1]
```diff
@@ -306,8 +306,8 @@ class CuSparse_SpmV_Pack<double,LocalOrdinal,GlobalOrdinal,Kokkos::Compat::Kokko
     cols   = reinterpret_cast<int*>(Acolind_cusparse.data());
     rowptr = reinterpret_cast<int*>(Arowptr_cusparse.data());
 
-    auto X_lcl = X.template getLocalView<device_type> ();
-    auto Y_lcl = Y.template getLocalView<device_type> ();
+    auto X_lcl = X.template getLocalViewDevice<device_type> ();
```
Comment (lucbv) [inline packages/muelu/test/scaling/MatvecKernelDriver.cpp:309]: Why not use this instead here: `getLocalViewDevice(Tpetra::Access::ReadWrite)` with no template parameters to make life easy : )

**Review by lucbv — CHANGES_REQUESTED** [review]
Just a small change to make maintenance easier down the road

File: packages/muelu/test/scaling/MatvecKernelDriver.cpp (hunk 1 above)
Comment (jhux2) [reply packages/muelu/test/scaling/MatvecKernelDriver.cpp]: Thanks, fixed.

**Review by lucbv — APPROVED** [review]
That looks good to me

**Pull request merged**